### ISSUE TITLE
React native web support

### ISSUE THREAD

**Issue by murtraja** [mmazzarolo/react-native-dialog#43]
This is an awesome library, but it lacks support for web. More specifically
  1. It doesn't have styling for web
  2. It crashes the web app as react-native-modal is not supported on web

#42 tries to fix 1. and provides config necessary for 2.

**Comment by murtraja** [comment]
The error is:
```
The above error occurred in the <ReactNativeModal> component:
    in ReactNativeModal (at Container.js:69)
    in DialogContainer (at App.js:305)
    in ErrorBoundary (at App.js:304)
    ...

  TypeError: "_index.DeviceEventEmitter is undefined"
```

**Comment by mmazzarolo** [comment]
@diego-antonelli published in [`v5.6.0`](https://github.com/mmazzarolo/react-native-dialog/releases/tag/v5.6.0)
